src/containers/bookForm.jsx

```diff
@@ -1,28 +1,78 @@
-import React from 'react';
-
-const BookForm = () => {
-  const categories = [
-    { id: 1, name: 'Action' },
-    { id: 2, name: 'Biography' },
-    { id: 3, name: 'History' },
-    { id: 4, name: 'Horror' },
-    { id: 5, name: 'Kids' },
-    { id: 6, name: 'Learning' },
-    { id: 7, name: 'Sci-Fi' },
-  ];
-  return (
-    <form>
-      <input type="text" placeholder="book title" />
-      <select>
-        {categories.map(c => (
-          <option key={c.id} value={c.name}>
-            {c.name}
-          </option>
-        ))}
-      </select>
-      <input type="submit" value="Save(Mileston 3)" />
-    </form>
-  );
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions';
+
+const mapDispatchToProps = dsipatch => ({
+  onSubmit: book => {
+    dsipatch(createBook(book));
+  },
+});
+
+class BookForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      book: { title: '', category: '' },
+    };
+    this.categories = [
+      { id: 0, name: ' ' },
+      { id: 1, name: 'Action' },
+      { id: 2, name: 'Biography' },
+      { id: 3, name: 'History' },
+      { id: 4, name: 'Horror' },
+      { id: 5, name: 'Kids' },
+      { id: 6, name: 'Learning' },
+      { id: 7, name: 'Sci-Fi' },
+    ];
```

Comment:
Since you will be using this in the next milestone in another file. I would recommend extracting it as a constant.

Reply:
Yes. That will be helpful. Proactive coding. :+1: